tensorboard/pip_package/build_pip_package.sh

```diff
@@ -114,6 +114,10 @@ build() (
   export PYTHONWARNINGS=ignore:DEPRECATION  # suppress Python 2.7 deprecation spam
   pip install -qU wheel 'setuptools>=36.2.0'
 
+  # Arbitrary date; overrides the file timestamps in the zip archive to
+  # make the build reproducible.
+  export SOURCE_DATE_EPOCH=1446856078
```

Comment:
This seems arbitrary to a confusing degree.  Can we just use 0?  This is usually the date I see for hermeticity purposes.  Or at least something that seems specifically chosen, like Jan 1 2020 at 00:00 UTC.

Reply:
I usually use 0, too, but avoided it to steer clear of bugs with Python,
Windows, and low timestamps, like <https://bugs.python.org/issue29097>.
I’m happy to switch this to `TZ=UTC date +%s -d '2020-01-01T00:00:00Z'`.

FWIW, this was the commit time of the first TensorBoard commit. :-)

Reply:
From the linked context it doesn't look like `datetime` is involved (just `time.gmtime()`), so `0` would seem safe enough to me and has the advantage of being self-explanatory in epoch timestamp format, but as you prefer.  Thanks!